benchmarkapps/Benchmarks/Program.cs

```diff
@@ -29,11 +29,11 @@ public static IWebHostBuilder GetWebHostBuilder(string[] args)
                     .UseKestrel();
 
             var scenario = config["scenarios"]?.ToLower();
-            if (scenario == "plaintextdispatcher" || scenario == "plaintextglobalrouting")
+            if (scenario == "plaintextdispatcher" || scenario == "plaintextglobalrouting" || scenario == "plaintextendpointrouting")
```

Comment:
You can nuke `plaintextglobalrouting`. It's not used anywhere yet. I was waiting for the dust to settle on this before making changes in the DB.